src/ChargeDrift/ChargeDrift.jl

```diff
@@ -287,20 +315,36 @@ function _drift_charge!(
     else
         zero(T)
     end
+    crystal_temperature::T = if diffusion
+        if haskey(det.semiconductor.material, :DT)
+            _parse_value(T, det.semiconductor.material.DT, u"K")
+        else 
+            @info "Ignoring diffusion for now"
+            diffusion = false
+            T(77)
+        end
+    else
+        T(77)
+    end

```

Comment:
There is a field in `det.semiconductor`, which is `temperature`.
I would suggest to not put the `temperature` in the material properties, but to pass it to the `Simulation` via the config file entry `temperature` in `semiconductor`.

The default temperature should then also be chosen there.

```suggestion
    crystal_temperature::T = det.semiconductor.temperature
```

The default for germanium, right now, is `78K`, whereas for everything else it is `293K`:
https://github.com/JuliaPhysics/SolidStateDetectors.jl/blob/32851dcd3d458879dfcebd07c2352dc681dddf06/src/SolidStateDetector/Semiconductor.jl#L76-L82

Reply:
I agree. Actually I have considered about this. I followed the practice in the temperature setting of the Boggs trapping model. 

So, I think we should unify all the temperatures in SSD.jl by reading from the config entry: `det.semiconductor.temperature`

Reply:
Ah, I see. I totally agree.